src/components/DocumentCard/DocumentCard.Props.ts

```diff
@@ -145,22 +146,10 @@ export interface IDocumentCardActionsProps extends React.Props<DocumentCardActio
   /**
    * The actions available for this document.
    */
-  actions: IDocumentCardAction[];
+  actions: IButtonProps[];
```

Comment:
instead of changing the type here, maybe you should leave the previous and deprecate it.

add "buttons" to the contract as an optional.

Then you are not breaking existing people. We need to avoid breaks as much as possible.